eduNEXT/xblock-imagesgallery
# fix: upload and delete files issue

### Description

When we attempted to upload new images and delete the previous ones, we encountered an error that resulted in the deletion of all images. This issue arose because the system was unable to perform both operations simultaneously. With this fix, it is now possible to execute both operations successfull 

BEFORE             |  NOW
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-imagesgallery/assets/134975835/cd6f05a2-aba3-464c-9197-9f89f0cf5991)  |  ![](https://github.com/eduNEXT/xblock-imagesgallery/assets/134975835/312f952f-fca8-4f42-8fc1-2aa9eded6451)


### How to test it

- Check this branch`jv/fix-delete-file-images` in the XBlock images gallery
1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 
```


**Merge checklist:**
Check off if complete *or* not applicable:
